enterprise/signals.py

```diff
@@ -47,4 +47,6 @@ def handle_user_post_save(sender, **kwargs):  # pylint: disable=unused-argument
         enterprise_customer=pending_link_record.enterprise_customer,
         user_id=user_instance.id
     )
+    for enrollment in pending_link_record.pendingenrollment_set.all():
```

Comment:
Would it be possible to rename `pending_link_record` to `pending_enterprise_customer_user`?  I'm finding the change in terminology a little confusing.